src/main/java/org/apache/cassandra/sidecar/MainModule.java

```diff
@@ -96,6 +97,12 @@ public Router vertxRouter(Vertx vertx, HealthService healthService)
 
         // API paths
         router.route().path("/api/v1/__health").handler(healthService::handleHealth);
+
+        router.route().path("/api/v1/compactions").handler(infoService::handleCompactions);
```

Comment:
Perhaps consider changing the path to `/api/v1/info/compactions`? That way all actions related to the `InfoService` are under the same path.